### ISSUE TITLE
ESI template contains unsupported detectInjection key value

### ISSUE THREAD

**Issue by mosborne6** [akamai/cli-property-manager#36]
When pulling down an existing config to create a new pipeline, it's created an ESI.json template based on the ESI rule as expected. However, validation checks warn of an unknown/unsupported value when saving the pipeline environment or attempting to promote it...

"type": "https://problems.luna.akamaiapis.net/papi/v0/validation/unknown_attribute",
        "errorLocation": {
            "template": "templates\\ESI.json",
            "variables": [],
            "location": "behaviors/0/options/detectInjection",
            "value": false
        },
        "detail": "The behavior '`ESI (Edge Side Includes)`' does not have detectInjection."

Removing this key:value pair from the JSON template resolves.
I'm guessing this may actually lie in Property Manager API, rather than the Pipeline itself?

Thanks!
![ESI JSON](https://user-images.githubusercontent.com/43175624/54369714-4b965100-466e-11e9-90cc-480777a86c6c.PNG)

**Comment by jnchin-james** [comment]
Yes, this might be an issue directly with property manager itself unfortunately.

**Comment by ddcrjlalumiere** [comment]
Having a very similar problem, despite these values already being set in the WebUI... Even just a noop `merge` with no changes produces this.

```
    {
        "type": "https://problems.luna.akamaiapis.net/papi/v0/validation/unknown_attribute",
        "errorLocation": {
            "template": "config-snippets/main.json",
            "variables": [],
            "location": "rules/behaviors/8/options/logEdgeIP",
            "value": false
        },
        "detail": "The behavior '`Log Request Details`' does not have logEdgeIP."
    },
```

**Comment by wzagrajcz** [comment]
Hello,
Can You please provide any details how to reproduce this issue? What configuration needs to be already present before running the command? What operation/command You run that produces the error?

Thanks and Regards,
Wojciech
